src/status_im2/contexts/onboarding/events.cljs

```diff
@@ -33,6 +36,18 @@
  (fn [request]
    (native-module/restore-account-and-login request)))
 
+(re-frame/reg-fx
+ :multiaccount/set-installation-name
+ (fn [installation-id]
+   (when platform/android?
+     (let [{:keys [model device-id]} (native-module/get-device-model-info)
+           device-name               (string/join " " [model device-id])]
```

Comment:
nit: we can make this a helper function in native-module, e.g native-module/get-device-name, or as part of the return value of `(native-module/get-device-model-info)`

Reply:
we sure can, but I don't see an obvious necessity in this